packages/dart_frog_cli/lib/src/daemon/exceptions.dart

```diff
@@ -27,3 +27,14 @@ class DartFrogDaemonMalformedMessageException
   const DartFrogDaemonMalformedMessageException(String message)
       : super('Malformed message, $message');
 }
+
+/// {@template dart_frog_daemon_missing_parameter_exception}
+/// An exception thrown when the daemon reports
+/// a missing parameter of a message.
```

Comment:
Nit: force doc to reach max length
```suggestion
/// An exception thrown when the daemon reports a missing parameter of a
/// message.
```